.deliveroo-local/make/generate-dockerfile.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# This script generates a Dockerfile adds deliveroo-local-certs to the base
+# Dockerfile. Ideally, we'd push out a base Routemaster image and fork from
+# there.
+
+dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+localdir="$( cd "${dir}/.." && pwd )"
+app="$( cd "${dir}/../../" && pwd )"
+
+# Add the certs image.
+cat << "EOF" > ${localdir}/Dockerfile
+FROM deliveroo/deliveroo-local-certs:latest as dev-certs
+EOF
+
+# List images.
+echo "$(head -n2 ${app}/Dockerfile)" >> ${localdir}/Dockerfile
+
+# Add certs.
+cat << "EOF" >> ${localdir}/Dockerfile
+
+# Add sudo.
+RUN apt-get update && apt-get install -y sudo && rm -rf /var/lib/apt/lists/*
+
+# Copy the certs from the deliveroo-local-ca image.
+COPY --from=dev-certs \
+     /usr/share/ca-certificates/deliveroo-local-ca.crt \
+     /usr/share/ca-certificates/deliveroo-local-ca.crt
+
+# Add the ca-cert to the store.
+RUN echo "deliveroo-local-ca.crt" >> /etc/ca-certificates.conf && \
+    sudo update-ca-certificates --fresh
```

Comment:
is `sudo` necessary here?

Reply:
Doesn’t seem like it! Removed this and the `sudo` installation in 2c552ee. Good catch! ❤